app/models/miq_server/environment_management.rb

```diff
@@ -26,12 +26,17 @@ def get_network_information
 
     def start_memcached
       # TODO: Need to periodically check the memcached instance to see if it's up, and bounce it and notify the UiWorkers to re-connect
```

Comment:
I wonder if we could hook into sync_config to check if the memcached config has changed so it isn't only on startup

Reply:
I don't know that we can change anything, other than these:

```
:session:
  :memcache_server: 127.0.0.1:11211
  :memcache_server_opts: "-l 127.0.0.1 -I 1M"
```

Even then, I think this is so infrequent that we might be able to detect these changes only on restart of the whole application.  🤔

Reply:
I'd be fine with a documentation note that says restart the whole application if you change memcached options or something

Reply:
Can we document how the user can modify the config file themselves instead of using our app to configure memcached?